variables.tf

```diff
@@ -30,6 +30,7 @@ variable "tfvars_s3_tfvars_files" {
   type = map(
     object({
       path = string
+      key  = optional(string, "")
       }
```

Comment:
can you fix this indentation please :)

Reply:
That's what `terraform fmt` sets it to for some reason